.tekton/ansible-ai-connect-service-pull-request.yaml

```diff
@@ -277,11 +277,6 @@ spec:
         - name: kind
           value: task
         resolver: bundles
-      when:
```

Comment:
IIUC you've removed the Snyk result check before applying the (quay) tags? Does this mean we'll effectively publish builds that failed analysis? 

Any reason for the change?

Reply:
Yes, for some reason, the check was broken and I spent the last 2h troubleshooting why the `pr-deployment` deployment was not working ^^.
This pipeline is only used to build the PR images, not for production nor stage. So I don't see any problem with tagging them with their `pr-XXX` label.

Reply:
Cc @jameswnl

Reply:
I will isolate the Konflux change in a dedicated PR.

Reply:
https://github.com/ansible/ansible-ai-connect-service/pull/1575